### ISSUE TITLE
Without --annotate output does not include RBI output

### ISSUE THREAD

**Issue by jasnow** [camertron/gelauto#3]
```
If you choose to run Gelauto without the --annotate flag, Gelauto will print results to standard output in RBI format and not modify any of your Ruby files.
```
I see the command output but nothing in RBI format.

**Comment by camertron** [comment]
This was a tough one to track down. Turns out the Ruby load path was getting overwritten. Fixed in 1.0.2.

**Comment by jasnow** [comment]
I would suggest you send this progress line to stderr to avoid merging them with the sig's.
```
I, [2019-07-08T12:54:05.614146 #32959]  INFO -- : [Gelauto] Indexed 19/222 paths
```
FYI: After editing the output, I got 1078 lines of sig's with a breakdown: 
* 89 class
* 296 def
*  97 end
* 296 sig

**Comment by camertron** [comment]
@jasnow good idea, and interesting breakdown. Is that what you expected?

**Comment by jasnow** [comment]
The tool runs cleanly. Next step is to experiment tool integration with my 9 sample repos. Do you have any Ruby or Rails versions restrictions?

**Comment by jasnow** [comment]
Closing this issue for now.

**Comment by camertron** [comment]
@jasnow it's being tested against ruby 2.5 and 2.6. You need at least Ruby 2.0 because that's the earliest version that supports the `TracePoint` API.

**Comment by camertron** [comment]
@jasnow just released v1.1.0 that logs to STDERR instead of STDOUT. Also added a global `--silent` flag and `--rbi` option for the `run` subcommand.

**Comment by jasnow** [comment]
Here is Sorbet's "What platforms does Sorbet support?" answer on this web page: https://sorbet.org/docs/faq
